app/src/main/java/com/simplecity/amp_library/utils/ThreadUtils.java

```diff
@@ -15,7 +14,6 @@ public static void ensureNotOnMainThread() {
             if (BuildConfig.DEBUG) {
                 throw new IllegalStateException("ensureNotOnMainThread failed.");
             } else {
-                CrashlyticsCore.getInstance().log("ThreadUtils ensureNotOnMainThread() failed");
```

Comment:
This shouldn't be an empty else statement - either remove else block or log this as an error like the others.